tests/Homely.AspNetCore.Mvc.Helpers.Tests/TestControllerTests/ModelBindingTests.cs

```diff
@@ -27,7 +34,7 @@ public async Task GivenABadModelBind_Get_ReturnsAnHttp400()
             // Arrange.
             var error = new ValidationProblemDetails
             {
-                //Type = "https://httpstatuses.com/400",
+                Type = "https://tools.ietf.org/html/rfc7231#section-6.5.1",

```

Comment:
is there no internal enum/const for this 'magic string'?

Reply:
This is defined in the .CORE sdk (undre problem details stuff)

[Here's an example](https://github.com/dotnet/aspnetcore/blob/00846a232c58db75c8477c9288af23508e055711/src/Mvc/Mvc.Core/src/DependencyInjection/ApiBehaviorOptionsSetup.cs#L62) of it getting used.

And [even their tests](https://github.com/dotnet/aspnetcore/blob/4a7bf75e89e3dfe7264a641ed041f50788d28d20/src/Mvc/test/Mvc.FunctionalTests/XmlDataContractSerializerFormattersWrappingTest.cs#L235) they reference the strings.

So i'm going to suggest - this might be ok for some test?